src/community/schemaless-features/schemaless-mongo/src/test/java/org/geoserver/schemalessfeatures/mongodb/WMSSchemalessMongoTest.java

```diff
@@ -260,4 +264,35 @@ private void checkStationFeature(JSONObject station) {
         assertNotNull(measurements);
         assertTrue(measurements.size() > 0);
     }
+
+    @Test
+    public void testSchemalessLayerInCapabilities() throws Exception {
```

Comment:
Please add a GetCapabilities test for WMS 1.3.0

Reply:
added